lib/githubactions/githubactions.go

```diff
@@ -118,3 +120,24 @@ func (c *IDTokenClaims) JoinAuditAttributes() (map[string]interface{}, error) {
 	}
 	return res, nil
 }
```

Comment:
Awkward comment placement but as in spacelift, is `JoinAuditAttributes` still used?

Reply:
ab2adae788